pkgs/by-name/yt/yt-dlp/package.nix

```diff
@@ -16,13 +16,13 @@ python3Packages.buildPythonApplication rec {
   # The websites yt-dlp deals with are a very moving target. That means that
   # downloads break constantly. Because of that, updates should always be backported
   # to the latest stable release.
-  version = "2024.8.6";
+  version = "2024.9.27";
```

Comment:
Do we really need the new websockets version for this update?

Reply:
You're right, relaxdeps is enough.